src/test/java/it/pn/frontend/e2e/stepDefinitions/common/HelpdeskTest.java

```diff
@@ -115,13 +119,25 @@ public void visualizzazioneCorrettaPaginaRicercaEdEstrazioneDati() {
         helpdeskPage.checkRicercaPage();
     }
 
+    @And("viene inserito codice fiscale {string}")
+    public void vieneInseritoCodiceFiscale(String nameFile) {
+        this.datiPersonaFisica = this.dataPopulation.readDataPopulation(nameFile+".yaml");
+        helpdeskPage.insertCfOnRicercaPage(datiPersonaFisica.get("codiceFiscale").toString());
+    }
+
+
     @When("Nella Home di helpdesk utente clicca su logout")
```

Comment:
in step definition non mettiamo @when ma un And  e Then  solo per i risultati

Reply:
done